src/challenges/challengeOne/1.Accordion/Accordion.test.tsx

```diff
@@ -1,8 +1,50 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Accordion } from "./Accordion";
-
-// describe("Accordion", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Accordion, AccordionProps } from "./Accordion";
+
+describe("Accordion", () => {
+  const renderComponent = (additionalProps?: AccordionProps) => {
```

Comment:
You'll want to use a `Partial` here so you can pass in a single prop without it throwing an error. 

```suggestion
  const renderComponent = (additionalProps?: Partial<AccordionProps>) => {
```

Reply:
Sure, Ben. I'll incorporate that.